### DISCUSSION TITLE
The suitablity of openage for orginal games

### DISCUSSION THREAD

**Discussion by springheeledjack66** [SFTtech/openage#1469] [General]
Hi, I'm a game development student and I've been looking to do projects outside of college and to network with people both professional and hobbyist, as such I wished to know if openage is a suitable game engine to produce an original game project.

**Comment by TheJJ** [comment]
I fear it is not suitable for you, since student projects usually have deadlines that would be earlier than I expect openage to be usable for you :)
You are probably better off with Godot, or - if necessary - lower level toolkits.

**Reply by springheeledjack66** [reply]
thank you though this isn't a college project it's just something I've been waiting to do as my course due to covid hasn't given me much in the field experience

**Reply by TheJJ** [reply]
Ok! You can surely find interesting parts to improve on, my recommendation to get into the project is to first build it on your machine, roughly understand the folder structure and have a look at the issues and the roadmap projects :)
